uk.co.compucorp.civicrm.hrleaveandabsences/js/angular/src/leave-absences/shared/components/leave-request-popup-details-tab.component.js

```diff
@@ -583,9 +525,6 @@ define([
 
       timeObject.loading = true;
```

Comment:
what was all this stuff used for?

Reply:
Before "loading" property was introduced it flushed values during data load, this is not needed anymore. The "loading" property was used to hide the time and deduction fields.